docker/models/services.py

```diff
@@ -124,6 +124,23 @@ def scale(self, replicas):
                                               service_mode,
                                               fetch_current_spec=True)
 
+    def restart(self):
```

Comment:
I think `restart` doesn't accurately represent the operation that's being performed here. In the Docker ecosystem, `restart` usually means "take the existing containers with their configuration and stop them then start them again". The same idea exists in other common software like nginx with the `reload` vs `restart` distinction. 

To my understanding. in this case, the containers for the service will actually be recreated with the same configuration - maybe `force_update` or `reload` is a more accurate name?